caseworker/views.py

```
@@ -0,0 +1,200 @@
+
+            summary = Summary.objects.create(
+                user=request.user,
+                case_number=case_number,
+                skill=skill,
+                content=content,
+            )
+            return Response(
+                self.get_serializer(summary).data, status=status.HTTP_201_CREATED
+            )
+
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)